src/main/java/technology/dice/dicefairlink/driver/AuroraReadDriver.java

```diff
@@ -36,37 +36,38 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-public class AuroraReadReplicasDriver implements Driver {
+public class AuroraReadDriver implements Driver {
```

Comment:
Still read-only. If this class name was to change (which it will not) we would also have to update the JNI service file

Reply:
Fixed.